### PR TITLE
Borers

### PR DESCRIPTION
**Pull request by oisin100**
Reduces Cortical Borers to 2 Commits. No more 130000 lines of code changed.

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
> borers done

:+1: x :100:

**Comment by Alek2ander** [comment]
:+1:

**Comment by oisin100** [comment]
Still need to create a changelog actually..

**Comment by ShadowDeath6** [comment]
Might want to hint at what they do in the changelog, it's a bit vague. Maybe add something like "Rumors are circulating that they are are some sort of parasite."

File: html/changelogs/Oisin100-Borers.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Nanotrasen has confirmed a new biohazard threat created by the <strike>nanotrasen</strike> syndicate animal experimentation labs"
```
Comment (AsV9) [inline html/changelogs/Oisin100-Borers.yml]: Hnnnnng `nanotrasen` in all lower-case. It should be `NanoTrasen` or `Nanotrasen` depending on how formal you want to be.

/ocd

EDIT: Also no punctuation. Arrrgh.

/ocd 4reals

Comment (oisin100) [inline html/changelogs/Oisin100-Borers.yml]: AYEE, SORRY MR FUHRER!

**Comment by Time-Green** [comment]
When will the server update and, all the features including boreres, be in the game?

**Comment by oisin100** [comment]
Also i should probably also mention that borers are a event. They can spawn any time after 20 minutes  and have the same spawning chances as Blob.

**Comment by Time-Green** [comment]
Same as blob?
Isn't that a little to small?
I have seen it 3 times in 5 months.

Can you rather make it like revenant? Maybe a little rarer?

**Comment by oisin100** [comment]
> Can you rather make it like revenant?

You mad bro? The same chance as revenant... So pretty much 100% chance of it happening every round?

How about half the chance of a electrical storm

File: html/changelogs/Oisin100-Borers.yml (hunk 1 above)
Comment (AsV9) [inline html/changelogs/Oisin100-Borers.yml]: it's `Mein Führer` you knob.

**Comment by AsV9** [comment]
@timkoster1 Implying not every single admin will adminbus them every round until people get bored.

**Comment by ShadowDeath6** [comment]
> BOREd

Blobs almost never spawn. I'd say give it the same chance as an anomaly or an ion law, maybe.

**Comment by ShadowDeath6** [comment]
Inb4 this gets merged but nobody updates for a week and a half

**Comment by oisin100** [comment]
Ion storm it is then. Anything else? Also it takes 5 minutes after using mind control till you can use it again.

Chems are:
`mannitol, bicardine, kelotane, charcoal, ephedrine, leporazine, morphine, space_drugs`

Should morphine be changed to tirizine or chloral with the latest morphine buffs?

All the medicines provide a 5% influence bonus on use whilst the morphine provides a -10% influence bonus

**Comment by Time-Green** [comment]
@AsV9 Wait? So I am allowed to spawn in mid-round antags if I feel like it?

**Comment by ShadowDeath6** [comment]
Tirizine is still way more powerful than morphine and chloral is still worse. I'd still say replace morphine with nulvoscine or whatever so it can mute the host but let them still do things.

**Comment by ghost** [comment]
Tirizine is better.

Morphine and chloral KO the borer as well as the host, fucking them over. Tirizine and add Pleufloradecalin as a muting chem.

**Comment by oisin100** [comment]
> Morphine and chloral KO the borer as well as the host, fucking them over.

Borer isnt affected by the hosts reagents(Except sugar)

Ill add Perfluorodecalin.

**Comment by ShadowDeath6** [comment]
Perflouro also works as a stabilizer too, but lets the host use *me, which is why I prefer nulvo. Also tirizine is fucking retardedly powerful as a KO chem, what are you smoking adam

**Comment by oisin100** [comment]
Using \* me to talk is against the rules unless you doing something like "*me-dances with ian"

**Comment by ShadowDeath6** [comment]
They can still do something like *me "points to head" or something.

**Comment by oisin100** [comment]
> They can still do something like *me "points to head" or something.

I dont mind that. Its not as if any doctor will ever understand that. And if they do well then the player got lucky.

**Comment by ShadowDeath6** [comment]
Fair enough.

**Comment by ShadowDeath6** [comment]
please merge ASAP the tension is killing me

**Comment by oisin100** [comment]
Borers now have perfluorodecalin instead of morphine.

Also fixed two bugs where borers werent space worthy and borers could increase their influence over 100% by spamming healing chems.

**Comment by ShadowDeath6** [comment]
Does it tell you how much influence you need when you try to do something you can't do without enough influence?

**Comment by AsV9** [comment]
I swear to god I thought github tracked renames. This conflicts with #441 because of tgstation.dme.
Ez to fix. Just do that, @Oisin100

**Comment by oisin100** [comment]
> Does it tell you how much influence you need when you try to do something you can't do without enough influence?

No it doesent.

> I swear to god I thought github tracked renames.

It does. But since this PR isnt up to date with the dev branch my branch still thinks tgstation.dme is called tgstation.dme

**Comment by oisin100** [comment]
Fuck, I accidentaly commited my Admins.txt file....

**Comment by ShadowDeath6** [comment]
""""Accidentally""""

> Oisin for council member when

**Comment by Cruix** [comment]
I'm telling Slov you tried to usurp him.

I usually use Overlord because it's cooler.

**Comment by ShadowDeath6** [comment]
Might want to spawn more borers in the event, like 2 or 3, since the first one is probably gonna get slam dunked/lynched before it can infest somebody or OD on sugar and get sliced out of the host's brain.

**Comment by oisin100** [comment]
> Might want to spawn more borers in the event, like 2 or 3, since the first one is probably gonna get slam dunked/lynched before it can infest somebody or OD on sugar and get sliced out of the host's brain.

Alrighty.

**Comment by ShadowDeath6** [comment]
_HYPE_

**Comment by AsV9** [comment]
How is this doing for logging? Admin buttons? Uhh. Check antags and stuff. Just curious because that's an absolute must for when you playtest this.

**Comment by oisin100** [comment]
> How is this doing for logging? Admin buttons? Uhh. Check antags and stuff. Just curious because that's an absolute must for when you playtest this.

Ah, forgot all about logging. Ill add some.

**Comment by oisin100** [comment]
Logging done Logs infection, spawning, reproducing, chem secretion and mindcontrolling to the game log. Admin buttons = Borer event can be starter from the "Trigger Event" verb.

**Comment by ShadowDeath6** [comment]
_**merge please**_

**Comment by AsV9** [comment]
Let's have a go then.

**Comment by ShadowDeath6** [comment]
## _**IT'S HAPPENING**_

**Pull request merged**